wagtail_wordpress_import/test/tests/utility_functions.py

```diff
@@ -0,0 +1,21 @@
+import tempfile
+from PIL import Image
+from bs4 import BeautifulSoup
+
+
+def get_soup(html, parser):
+    soup = BeautifulSoup(html, parser)
+    return soup
+
+
+def mock_image():
+    temp_file = tempfile.NamedTemporaryFile(suffix=".jpg")
+    image = Image.new("RGB", (200, 200), "white")
+    image.save(temp_file, "PNG")
+    return open(temp_file.name, mode="rb")
```

Comment:
The issue is fixed in 98c1cfe.

Reply:
The image tempfile issue is fixed in 2a2278d.

I meant that this is an issue for all functions using tempfile, but probably it can be left if you prefer. In general the use of the filesystem is nice to avoid if possible during unit tests, unless you _need_ to test that something is correctly read or written.

Reply:
@nickmoreton Please respond, or resolve this thread, as necessary.